src/views/email.njk

```diff
@@ -1,5 +1,9 @@
 {% extends "layout.njk" %}
 
+{% block pageTitle %}
```

Comment:
it might be worth creating a common component automatically included on each page
and only set the title from the page

Reply:
It's a very small block, the benefits would be limited.

Reply:
it's to avoid repeating this part every time "- {{ i18n.service }} - GOV.UK" and avoid mistakes
something like the back-link

Reply:
Ok I have placed this into it's own includes file.

Reply:
better, but you might as well include the block too so that you only get the includes on the page

Reply:
The does not work, it has to be declared as a block in the template itself.

Reply:
Moved block to layout.

Reply:
the other pages are correct, but this one still contains the block pageTitle

Reply:
Saw that and have removed it.